snuba/admin/static/runtime_config/auditlog.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactNode, useState } from "react";
+
+import Client from "../api_client";
+import { containerStyle, paragraphStyle } from "./styles";
+import { Table } from "../table";
+import { ConfigValue } from "./types";
+
+function AuditLog(props: { api: Client }) {
+  const { api } = props;
+  const [data, setData] = useState<any[] | null>(null);
+
+  function fetchData() {
+    api.getAuditlog().then((res) => {
+      setData(res);
+    });
+  }
+
+  if (data === null) {
+    fetchData();
+  }
+
+  if (!data) {
+    return null;
+  }
```

Comment:
should this be in `constructor`? I find the control flow a bit confusing here but maybe this is a react style that I'm not accustomed to.

Reply:
In class based React, I think this is normally in `componentDidMount` as async things don't normally happen in the constructor. In hooks based React, I think this is often inside a useEffect hook. But this seemed the easiest way to only run fetch once.